src/types/sigstore/index.ts

```diff
@@ -34,7 +35,14 @@ export * from './__generated__/sigstore_trustroot';
 export * from './__generated__/sigstore_verification';
 
 export const bundleToJSON = Bundle.toJSON;
-export const bundleFromJSON = Bundle.fromJSON;
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const bundleFromJSON = (obj: any): ValidBundle => {
+  const bundle = Bundle.fromJSON(obj);
+  assertValidBundle(bundle);
+  return bundle;
+};
```

Comment:
Adding the call to `assertValidBundle` here means that we'll automatically validate the bundle at the point that we deserialize it from JSON. If this function returns, it is guaranteed to return a `ValidBundle`.

Reply:
Nice! Types FTW 🙌